examples/src/main/scala/org/apache/spark/examples/ml/ALSExample.scala

```diff
@@ -28,7 +28,7 @@ object ALSExample {
 
   // $example on$
   case class Rating(userId: Int, movieId: Int, rating: Float, timestamp: Long)
-  object Rating {
+  object RatingUtil {
```

Comment:
I can move it into the main. I think it is not necessary. I will make the change and test it.

Thanks!